backend/app/services/execution/router.py

```
@@ -0,0 +1,164 @@
+        },
+        "Agno": {
+            "mode": "http_callback",
+            "url": None,
+            "endpoint": "/v1/execute",
+        },
+        "Dify": {
+            "mode": "http_callback",
+            "url": None,
+            "endpoint": "/v1/execute",
+        },
+    }
```

Ruff (0.14.14)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)